galsim/interpolatedimage.py

```diff
@@ -25,6 +25,7 @@
 from . import _galsim
 from ._galsim import Interpolant
 from ._galsim import Nearest, Linear, Cubic, Quintic, Lanczos, SincInterpolant, Delta
+import numpy as np
```

Comment:
I don't believe so.  I think python only actually reads in the files the first time a module is imported.  After that it's just about letting that namespace be visible in the current scope.